src/openforms/formio/serializers.py

```diff
@@ -87,6 +87,9 @@ def _remove_validations_from_field(self, field: serializers.Field) -> None:
                 field.min_length = None
                 field.max_length = None
 
+            case serializers.ChoiceField():
+                field.allow_blank = True
+
```

Comment:
Can't we just have a way to disable validation completely if the field is hidden? This feels like a hacky way of making validation pass

Reply:
no, because the datatype still needs to match and the JSON input needs to be converted to the appropriate python type.